test/e2e/simple.go

```diff
@@ -15,7 +15,9 @@ import (
 // and MsgSends over 30 seconds and then asserts that at least 10 transactions were
 // committed.
 func E2ESimple(logger *log.Logger) error {
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
why do you need a cancellable context here? is it getting cancelled somewhere other than the end of the function call?

Reply:
good point. will fix it.